### PR TITLE
[TW] Initial zh-tw translation

### PR DESCRIPTION
**Pull request by lithiumpie**
@doomspork @cizixs 
Initial zh-tw translation 1. Basics/2. Collections/3. Enum/4. Pattern Matching/5. Control Structures/6. Functions/7. Pipe Operator/8. Modules/9. Mix/10. Sigils

Include Index.md and /_data/locales/tw.yml

### CODE REVIEW

**Comment by doomspork** [comment]
@lithiumpie this is amazing, thank you so much!  What is your twitter handle?

**Comment by lithiumpie** [comment]
[@Lithium_Pie](https://twitter.com/Lithium_Pie) Hola, amigo:)

**Comment by cizixs** [comment]
@lithiumpie Truly amazing work!

@doomspork This is **Traditional Chinese** translation, and I'm working on **Simplified Chinese**, since there is no reviewer for this commit right now, and the two languages are similar. I volunteer to review this commit for now. What do you think?

**Comment by doomspork** [comment]
@cizixs that would be wonderful!  Should I create a separate translation team for these translations or do you want them to all fall under @elixirschool/chinese ?

**Comment by lithiumpie** [comment]
I like simple things and life should be simple. Here is my opinion. I do not really know how team elixirschool/chinese works but I also do not care just under elixirschool/chinese if it is possible to label zh-cn and zh-tw. Traditional Chinese and Simplified Chinese is just like British English and American English. Most of them are the same. Culture and glossary will be major difference. For example, operator will translate into 運算子 (zh-tw) but 運算符 (zh-cn).

**Comment by cizixs** [comment]
@doomspork I think it's better we can create a separate translation team since these two languages are different enough for me to handle. Secondly, I hardly have so much time to review all the PRs.

@lithiumpie If you have the time and determination to continue translating, and are willing to review and tutor all the translators later-on, we call name you in charge of the new team.

File: tw/lessons/basics/basics.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+```elixir
+iex> true
+true
+iex> false
+false
+```
+
+### Atoms
+
+Atom 是一個常數，其名稱是它的值。如果你熟悉 Ruby，這些符號是同義詞：
```
Comment (cizixs) [inline tw/lessons/basics/basics.md]: 最后一句是不是 typo？应该是 `这和符号是同义词`

**Comment by cizixs** [comment]
@lithiumpie I have read the PR though and found a typo, please check if a change should be made or I made a mistake. 

The rest of them are excellent. And I'll upvote it soon.

**Comment by doomspork** [comment]
@cizixs I'm happy to add others to the translation team to ease your burden.  Do you have suggestions?  😀

File: tw/lessons/basics/basics.md (hunk 1 above)
Comment (lithiumpie) [reply tw/lessons/basics/basics.md]: **EN:** If you’re familiar with Ruby, these are synonymous with Symbols:
**CN:** 如果你熟悉 Ruby，它们和Ruby中的符号是同义的。

@cizixs Yep, you're right. I will revise the translation

**TW:** 如果你熟悉 Ruby，這與 Ruby 的符號 (Symbols) 是同義詞：

**Pull request merged**